### PR TITLE
[Legacy Platform] Ft load map nodes

### PR DESCRIPTION
**Pull request by FaithDaka**
#### Summary of Changes (What does this PR do?)

- Allows redux to fetch map data outside user login

#### Status of maturity (all need to be checked before merging):

- [x] I've tested this locally
- [x] I consider this code done
- [x] This change ready to hit production in its current state
- [x] The title of the PR states what changed and the related issues number (used for the release note).

#### How should this be manually tested?
- View deploy preview

#### What are the relevant tickets?
- [AN-248]

#### Screenshots 
image

[AN-248]: https://airqoteam.atlassian.net/browse/AN-248?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Baalmart — APPROVED** [review]
Thanks @FaithDaka , this looks good to me.

**Review by Codebmk — COMMENTED** [review]
Running this locally. Loaded for the first time and got the brown nodes. On second load, it was normal.
Screenshot from 2023-01-02 14-51-19

**Review by Baalmart — COMMENTED** [review]
Hi @FaithDaka ,

On second review, I have observed that the nodes are brown on first load.

And all is okay after reload.....

**Comment by FaithDaka** [comment]
@Codebmk @Baalmart That has been resolved!

**Comment by FaithDaka** [comment]
@Codebmk @Baalmart Should be okay now!

**Comment by FaithDaka** [comment]
@Codebmk @Baalmart Should be okay now!

File: netmanager/src/redux/Join/actions.js [hunk 1]
```diff
@@ -67,39 +67,46 @@ import {
...
 /***************************organization actions ********************************* */
 export const setOrganization = () => (dispatch, getState) => {
-  const name = getState().auth.user.organization
-  dispatch(updateOrganization({ name }))
-}
+  const name = getState().auth.user.organization;
+  if (name) {
+    dispatch(updateOrganization({ name }));
+  } else {
+    dispatch(updateOrganization('airqo'));
```
Comment (Baalmart) [inline netmanager/src/redux/Join/actions.js:86]: thanks @FaithDaka 

In the near future, we shall only be using one tenant (AirQo) in our system.
All other networks will be present under the AirQo tenant.

cc: @Codebmk

**Review by Baalmart — APPROVED** [review]
Thanks @FaithDaka , this looks good to me.

**Pull request merged**